meson.build

```diff
@@ -119,12 +119,20 @@ if cpu_family == 'x86'
 
   # Need to link against libatomic for 64-bit atomic emulation on x86
   # when using clang.
-  if vkd3d_clang
+  if vkd3d_is_clang
     lib_atomic = vkd3d_compiler.find_library('atomic')
     vkd3d_extra_libs += lib_atomic
   endif
 endif
 
+if not vkd3d_is_msvc
+  # We need to set the section alignment for debug symbols to
+  # work properly as well as avoiding a memcpy from the Wine loader.
+  if vkd3d_compiler.has_link_argument('-Wl,--file-alignment=4096')
+    add_global_link_arguments('-Wl,--file-alignment=4096', language: 'cpp')
```

Comment:
vkd3d-proton uses C as well

Reply:
Resolved.

Reply:
The linker has to use C++ due to dxil-spirv though.

Reply:
I set it for both C and C++